### ISSUE TITLE
Why I still use 'Yggdroot/indentLine' ?

### ISSUE THREAD

**Issue by jonahfang** [lukas-reineke/indent-blankline.nvim#633]
(1) I can use line style like this:

<img width="435" alt="image" src="https://github.com/lukas-reineke/indent-blankline.nvim/assets/16301509/388554a2-5a69-4d6d-94c6-0cecbd0817d3">

(2) The indent lines begin at second indent level by default:

<img width="783" alt="image" src="https://github.com/lukas-reineke/indent-blankline.nvim/assets/16301509/5b4a3e64-e6fd-4f82-b698-5158388d164c">

**Comment by jonahfang** [comment]
I use config as follows  and finally use this plugin again:

```
  config = function()
    local hl_list = {}
    -- hide first level line by color (#1E1E2E)
    local c = '#2A2B3C'
    for i, color in pairs({ '#1E1E2E',c,c,c,c,c,c,c,c,c,c,c,c,c}) do
      local name = 'IndentBlanklineIndent' .. i
      vim.api.nvim_set_hl(0, name, { fg = color, bg = color })
      table.insert(hl_list, name);
    end
    require('indent_blankline').setup {
      char_highlight_list = hl_list,
    }
  end,

```
effect like this:
![ok](https://github.com/lukas-reineke/indent-blankline.nvim/assets/16301509/290d6e04-fc69-43b1-9f2d-ce4e74ca7fc3)

**Comment by siduck** [comment]
```lua
      local hooks = require "ibl.hooks"
      hooks.register(hooks.type.WHITESPACE, hooks.builtin.hide_first_space_indent_level)
```

**Comment by jonahfang** [comment]
@siduck , Thanks for reply. How to install "ibl.hooks"?

**Comment by siduck** [comment]
> @siduck , Thanks for reply. How to install "ibl.hooks"?

blankline recently released v3.0 so the config you wrote is actually for old version!

just install the plugin normally and use this as config

```lua
      local hooks = require "ibl.hooks"
      hooks.register(hooks.type.WHITESPACE, hooks.builtin.hide_first_space_indent_level)

      require("ibl").setup()
```

**Comment by siduck** [comment]
Also read https://github.com/lukas-reineke/indent-blankline.nvim for new features!
